datafusion/core/tests/sql/window.rs

```diff
@@ -578,6 +578,264 @@ async fn window_frame_rows_preceding_with_partition_unique_order_by() -> Result<
+               ORDER BY c9 \
+               LIMIT 5";
+    let actual = execute_to_batches(&ctx, sql).await;
+    let expected = vec![
+        "+----------------------------+----------------------------+-----------------+",
+        "| SUM(aggregate_test_100.c4) | SUM(aggregate_test_100.c3) | COUNT(UInt8(1)) |",
+        "+----------------------------+----------------------------+-----------------+",
+        "| 52276                      | 781                        | 56              |",
+        "| 260620                     | 781                        | 63              |",
+        "| -28623                     | 781                        | 37              |",
+        "| 260620                     | 781                        | 63              |",
+        "| 260620                     | 781                        | 63              |",
+        "+----------------------------+----------------------------+-----------------+",
+    ];
+    assert_batches_eq!(expected, &actual);
+    Ok(())
+}
+
+#[tokio::test]
+async fn window_frame_order_by_asc_desc_large() -> Result<()> {
+    let ctx = SessionContext::new();
+    register_aggregate_csv(&ctx).await?;
+    let sql = "SELECT
+                SUM(c5) OVER (ORDER BY c2 ASC, c6 DESC)
+                FROM aggregate_test_100
+                LIMIT 5";
+    let actual = execute_to_batches(&ctx, sql).await;
+    let expected = vec![
+        "+----------------------------+",
+        "| SUM(aggregate_test_100.c5) |",
+        "+----------------------------+",
+        "| -1383162419                |",
+        "| -3265456275                |",
+        "| -3909681744                |",
+        "| -5241214934                |",
+        "| -4246910946                |",
+        "+----------------------------+",
+    ];
+    assert_batches_eq!(expected, &actual);
+    Ok(())
+}
+
+#[tokio::test]
+async fn window_frame_order_by_desc_large() -> Result<()> {
+    let ctx = SessionContext::new();
+    register_aggregate_csv(&ctx).await?;
+    let sql = "SELECT
+                SUM(c5) OVER (ORDER BY c2 DESC, c6 ASC)
+                FROM aggregate_test_100
+                ORDER BY c9
+                LIMIT 5";
+    let actual = execute_to_batches(&ctx, sql).await;
+    let expected = vec![
+        "+----------------------------+",
+        "| SUM(aggregate_test_100.c5) |",
+        "+----------------------------+",
+        "| 11212193439                |",
+        "| 22799733943                |",
+        "| 2935356871                 |",
+        "| 15810962683                |",
+        "| 18035025006                |",
+        "+----------------------------+",
+    ];
+    assert_batches_eq!(expected, &actual);
+    Ok(())
+}
+
+#[tokio::test]
+async fn window_frame_order_by_null_timestamp_order_by() -> Result<()> {
+    let ctx = SessionContext::new();
+    register_aggregate_null_cases_csv(&ctx).await?;
+    let sql = "SELECT
+                SUM(c1) OVER (ORDER BY c2 DESC)
```

Comment:
I think `sum` will produce the same value regardless of the window order -- perhaps using `first()` might add better coverage to these tests